include/zephyr/bluetooth/gatt.h

```diff
@@ -296,8 +296,13 @@ struct bt_gatt_attr {
 	 *  satisfied before calling read() or write().
 	 *
 	 *  @sa bt_gatt_discover_func_t about this field.
+	 *
+	 *  @note The `_auto` flag in this bitfield indicates if the attribute handle was
+	 *  automatically assigned by the stack (1) or manually set by the application (0).
```

Comment:
```suggestion
	 *  automatically assigned by the stack (1) or manually set by the application (0). It must not be modified by the application.
```

Reply:
I suggest adding `@internal` and saying it's private.